server/form-pages/assess/matchingInformation/matchingInformationTask/matchingInformation.ts

```diff
@@ -107,7 +100,6 @@ export default class MatchingInformation implements TasklistPage {
   get body(): MatchingInformationBody {
     return {
       ...this._body,
-      accessibilityCriteria: this._body.accessibilityCriteria ? [this._body.accessibilityCriteria].flat() : [],
       specialistSupportCriteria: this._body.specialistSupportCriteria
```

Comment:
This breaks a couple of tests. I'm not sure I understand the need around these tests, but it might be something to pick up separate to this PR in any case?

```
● MatchingInformation › body › should return specialistSupportCriteria as arrays if strings are provided
● MatchingInformation › body › should return empty arrays for specialistSupportCriteria as arrays if no data is provided
```